openapi.yaml

```diff
@@ -47,33 +47,45 @@ info:
     | **HTTP Authorization Scheme** | bearer |
 
     ## OAuth
+    If all you need is a Personal Access Token, feel free to skip ahead to the next section.
 
-    The OAuth workflow is a three-step process to authenticate a User before an
-    application can start making API calls on the User's behalf. If all you need
-    is a Personal Access Token, feel free to skip ahead to the next section.
+    For more help on OAuth see our guide on [How to Create an OAuth App with teh Linode Python API Library](https://www.linode.com/docs/platform/api/how-to-create-an-oauth-app-with-the-linode-python-api-library/#oauth-2-authentication-exchange).
 
-    First, the User visits the application's website and is directed to log with
-    Linode. The User is then redirected to Linode's authentication server and
-    presented the scope levels the application is requesting. Once the User
-    accepts the request for access, we redirect them back to the application's
-    specified redirect URI with an access code.
+    Before you begin, you need to [Create an OAuth Client](https://developers.linode.com/api/v4/account-oauth-clients/#post).
```

Comment:
Might also consider https://cloud.linode.com/profile/clients

Reply:
changed to include both options